com.reprezen.swagedit.dependencies/pom.xml

```diff
@@ -53,22 +78,71 @@
 						<goals>
 							<goal>copy-dependencies</goal>
 						</goals>
+						<configuration>
+							<outputDirectory>${project.basedir}/lib</outputDirectory>
+							<overWriteReleases>false</overWriteReleases>
+							<overWriteSnapshots>true</overWriteSnapshots>
+							<includeTypes>jar</includeTypes>
+							<prependGroupId>true</prependGroupId>
+							<stripVersion>true</stripVersion>
+							<excludeArtifactIds>com.reprezen.swagedit.dependencies</excludeArtifactIds>
+							<excludeGroupIds>
+								com.google.guava,
+								javax.mail,
+								javax.activation,
+								com.sun.mail
+							</excludeGroupIds>
+							<excludeScope>system</excludeScope>
+							<includeScope>runtime</includeScope>
+						</configuration>
+					</execution>
+					<execution>
+						<id>classpath</id>
+						<goals>
+							<goal>build-classpath</goal>
+						</goals>
+						<configuration>
+							<prefix>lib</prefix>
+							<prependGroupId>true</prependGroupId>
+							<outputFile>${project.basedir}/lib/classPath.txt</outputFile>
+							<excludeArtifactIds>com.reprezen.swagedit.dependencies</excludeArtifactIds>
+							<excludeScope>system</excludeScope>
+							<includeScope>runtime</includeScope>
+						</configuration>
 					</execution>
 				</executions>
-				<configuration>
-					<outputDirectory>${project.basedir}/lib</outputDirectory>
-					<overWriteReleases>false</overWriteReleases>
-					<overWriteSnapshots>true</overWriteSnapshots>
-					<includeTypes>jar</includeTypes>
-					<prependGroupId>true</prependGroupId>
-					<excludeGroupIds>
-						com.google.guava,
-						javax.mail,
-						javax.activation,
-						com.sun.mail
-					</excludeGroupIds>
-				</configuration>
 			</plugin>
 		</plugins>
+
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>org.eclipse.m2e</groupId>
+					<artifactId>lifecycle-mapping</artifactId>
+					<version>1.0.0</version>
+					<configuration>
+						<lifecycleMappingMetadata>
+							<pluginExecutions>
+								<pluginExecution>
+									<pluginExecutionFilter>
+										<groupId>org.apache.maven.plugins</groupId>
+										<artifactId>maven-dependency-plugin</artifactId>
+										<versionRange>[2.8,)</versionRange>
+										<goals>
+											<goal>build-classpath</goal>
+										</goals>
+									</pluginExecutionFilter>
+									<action>
+										<execute>
+											<runOnIncremental>false</runOnIncremental>
```

Comment:
OK, this whole `<pluginManagement>` construct was added to prevent execution of the `build-classpath`  goal each time there's an incremental build in Eclipse, right?

Reply:
That's specific to the maven eclipse plugin. It's to make sure the `build-classpath` goal will be executed by maven inside eclipse.  Otherwise it may not do it and you will have to build the project or execute that goal manually to see it executed in eclipse.